scripts/test.sh

```diff
@@ -1 +1,13 @@
+if [ -n "${TRAVIS_BRANCH+1}" ]; then
```

Comment:
If we are on travis, we need travis to start up the redis server to its configuration. If we're running locally, we'll have our test command start up a server for us. ```-n "${TRAVIS_BRANCH+1}"``` checks to see if TRAVIS_BRANCH is defined in the shell script.